pkgs/development/python-modules/pytest-xdist/1.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchPypi, buildPythonPackage, execnet, pytest
+, setuptools_scm, pytest-forked, filelock, psutil, six, isPy3k }:
+
+buildPythonPackage rec {
+  pname = "pytest-xdist";
+  version = "1.34.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1vh4ps32lp5ignch5adbl3pgchvigdfmrl6qpmhxih54wa1qw3il";
+  };
+
+  nativeBuildInputs = [ setuptools_scm pytest ];
+  checkInputs = [ pytest filelock ];
```

Comment:
Can we use pytestCheckHook here?

Reply:
The diff is deceiving. I did not write this but rather renamed default.nix from before https://github.com/NixOS/nixpkgs/commit/6c1eec9838ab438f5120878ca579db7d2753cdba to 1.nix. This pull request is not about refactoring; it is only about fixing the build.